app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -225,7 +225,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-    private void startFeedDownload(String url, String username, String password) {
+    private void startFeedDownload(String url, String username, String password, boolean initiatedByUser) {
```

Comment:
This parameter seems to be always true. You can just remove it :)

Reply:
Fixed.